Unix/configeditor/configeditor.cpp

```diff
@@ -423,13 +446,30 @@ int MI_MAIN_CALL main(int argc, const char** argv)
             }
             ss << commentByte << prop << "=" << _CreateStringFromList(valueString,valueList);
         }
+        // Support querying a value from a property list
+        else if (!opts.query.empty())
+        {
+        	std::list<std::string> valueList;
+        	
+        	_ParseValueList(valueList, value);
+        	if (_IsValueInList(valueList, opts.query) != valueList.end())
+        	{
+        	    cout << prop << " have the value " << opts.query << std::endl;
+        	    return 0;
+        	}
+                cout << prop << " does not have the value " << opts.query << std::endl;
+        	return 1;
+        }
         else
         {
             // Um, what are we supposed to do?
             err(ZT("Logic error, file %s, line %d"), scs(__FILE__), __LINE__);
         }
-
-        cout << ss.str() << std::endl;
+        
```

Comment:
Needless spaces in here. Blank line is fine (and good), prefer blank line without spaces.

A project I'm working on, mesos, actually as a git commit validity checker that would reject commits that have this sort of thing ...